packages/teams-js/src/private/menus.ts

```diff
@@ -89,7 +89,7 @@ export namespace menus {
      * @hidden
      * Interface to show list of items on selection of menu item.
      */
-    public viewData: ViewData;
+    public viewData?: ViewData;
```

Comment:
Is this an intentional change?

Reply:
Yes, I may be misunderstanding this API, but it looks like this should be an optional property. If `viewData` is a required property of `menuItem` and `menuItems` is a required property of `viewData` then it seems like it would be impossible to have some terminal menu item that doesn't open another menu.

Reply:
Got it thanks